AppCenter/AppCenterTests/MSOneCollectorChannelDelegateTests.m

```diff
@@ -324,19 +328,38 @@ - (void) testDoesNotFilterLogFromOneCollectorChannel {
                                                                                 batchSizeLimit:1024
                                                                            pendingBatchesLimit:60];
   OCMStub([oneCollectorChannelUnitMock configuration]).andReturn(unitConfig);
-  NSMutableSet *transmissionTargetTokens = [NSMutableSet new];
-  [transmissionTargetTokens addObject:@"fake-transmission-target-token"];
-  id<MSLog> mockLog = OCMProtocolMock(@protocol(MSLog));
-  OCMStub(mockLog.transmissionTargetTokens).andReturn(transmissionTargetTokens);
+  MSCommonSchemaLog *log = [MSCommonSchemaLog new];
+  log.name = @"avalidname";
 
   // When
-  BOOL shouldFilter = [self.sut channelUnit:oneCollectorChannelUnitMock shouldFilterLog:mockLog];
+  BOOL shouldFilter = [self.sut channelUnit:oneCollectorChannelUnitMock shouldFilterLog:log];
 
   // Then
   XCTAssertFalse(shouldFilter);
 }
 
-- (void) testDoesNotFilterLogFromNonOneCollectorChannelWhenLogHasNoTargetTokens {
+- (void)testFilterInvalidCommonSchemaLogs {
```

Comment:
super nit: `testFilter`->`testFilters` would be more consistent with other names